openc3/lib/openc3/migrations/20241107120000_no_trigger_group.rb

```diff
@@ -0,0 +1,29 @@
+require 'openc3/utilities/migration'
+require 'openc3/models/scope_model'
+require 'openc3/models/microservice_model'
+
+module OpenC3
+  class NoTriggerGroups < Migration
+    begin
+      require 'openc3-enterprise/models/cmd_authority_model'
+      require 'openc3-enterprise/models/critical_cmd_model'
+      BASE = false
+    rescue LoadError
+      BASE = true
+    end
+
+    def self.run
+      if BASE # Only remove the trigger group microservice if we're not enterprise
+        MicroserviceModel.get_all_models(scope: 'DEFAULT').each do |microservice_name, microservice_model|
```

Comment:
Don't we need to loop through all the scopes like in `no_critical_cmd`?

Reply:
Base only has DEFAULT scope